tasks/create-pyxis-image/tests/mocks.sh

```diff
@@ -9,7 +9,7 @@ function create_container_image() {
   # e.g. 0001, 0002, 0003...
   echo The image id is $(awk 'END{printf("%04i", NR)}' $(workspaces.data.path)/mock_create_container_image.txt)
 
-  if [[ "$*" != "--pyxis-url https://pyxis.preprod.api.redhat.com/ --certified false --tags "*" --is-latest false --verbose --oras-manifest-fetch /tmp/oras-manifest-fetch.json --name "*" --media-type my_media_type --digest "*" --architecture-digest "*" --architecture "*" --rh-push "* ]]
+  if [[ "$*" != "--pyxis-url https://pyxis.preprod.api.redhat.com/ --certified false --tags "*" --is-latest false --verbose --oras-manifest-fetch ${MANIFEST_FILE} --name "*" --media-type my_media_type+gzip --digest "*" --architecture-digest "*" --architecture "*" --rh-push "* ]]
```

Comment:
Did you get the output of the tekton task unit tests workflow to make sure this is behaving as you'd expect? Just not used to seeing a variable from the task in the mocks file (if it isn't working right, you could always `s/${MANIFEST_FILE}/"*"/`)

Reply:
Yeah, it was suggested by @mmalina.

You know, it makes sense because the mocks are literally prepended to the step scripts in the pre apply hook.

But, anyways, it occurs to me that we could just replace it here with `*` to keep it simple and unsurprising.

Reply:
Dropped in 5babb419c22301b7d6991204e3f4bec018a41bff.